src/core/utils.js

```diff
@@ -86,9 +86,6 @@ export function fromJSOrdered(js) {
     const objWithHashedKeys = createObjWithHashedKeys(js)
```

Comment:
The logic here is a little bit nonsensical for me here.

`createObjWithHashedKeys` is used in only one place in codebase, and that's here on line 86. 

Then first thing the function does is validate it's input param again but with different predicate statement. This seems redundant and non-compatible with `isFunction(js.entries)` predicate statement checking if `entries` is a function instead checking if it exists. I recommend dropping following code from the function body.

```js
 if (!fdObj.entries) {
    return fdObj // not a FormData object with iterable
  }
```

Another thing about `createObjWithHashedKeys` is that it's super crazy complicated. It's not apparent what is does by reading it. So I'd either put `@example` tag to functions existing jsdoc to document what is the expected output of this function for particular input (formdata) or create a unit tests for this function.

Reply:
> This seems redundant  

Internal validation updated to match `isFunction` for consistency. Internal validation retained for robustness

> put @example tag

Added a real example

> unit tests

Not done for 'createObjWithHashedKeys'. Reason: would be new feature with additional dependencies for isomorphic-form-data for testing purposes.

However, there exists a `curlify` test that tests rendering of the result, which is basis of the added @example tag mentioned above

Reply:
OK everything clear except one thing.

We're trying to detect if the `js` is a FormData instance. Why are we not using `instanceof FormData` operator but rather asserting on the shape of the possible formData instance? `swagger-client` is integral part of swagger-ui and it makes sure that `FormData` symbol is always present as global symbol, both in browser and node.js so it's absolutely safe to do so without installing `isomorphic-form-data` explicitly. 

> Reason: would be new feature with additional dependencies for isomorphic-form-data for testing purposes

New feature? I'd rather say it's just a dev-dependency used only for tests. But anyway @examples in jsdoc should suffice if function is tested elsewhere in integration.